### PR TITLE
UCP/CORE: Added VFS nodes representing UCP listener.

### PR DESCRIPTION
**Pull request by rakhmets**
## What
Added directory to VFS tree representing UCP listener object.
Added UCP listener ip address and port to VFS structure.

## Why ?
The changes help to perform runtime analysis of UCX-based applications.

### CODE REVIEW

File: src/ucp/core/ucp_listener.c [hunk 1]
```diff
@@ -246,6 +247,36 @@ ucp_listen(ucp_listener_h listener, const ucp_listener_params_t *params)
     return status;
 }
 
+static void ucp_listener_vfs_show_ip(void *obj, ucs_string_buffer_t *strb)
+{
+    ucp_listener_h listener   = obj;
+    struct sockaddr *sockaddr = (struct sockaddr*)&listener->sockaddr;
+    char ip_str[UCS_SOCKADDR_STRING_LEN];
+
+    if (inet_ntop(sockaddr->sa_family, ucs_sockaddr_get_inet_addr(sockaddr),
```
Comment (brminich) [inline src/ucp/core/ucp_listener.c:256]: 1. please compare with `NULL` explicitly.
2. maybe, if `inet_ntop` fails, printing the errno makes sense?

File: src/ucp/core/ucp_listener.c [hunk 2]
```diff
@@ -246,6 +247,38 @@ ucp_listen(ucp_listener_h listener, const ucp_listener_params_t *params)
...
```
Comment (yosefe) [inline src/ucp/core/ucp_listener.c:256]: i'd suggest to add helper function fo sock.h/c to return the string of net part: ucs_sockaddr_get_ipstr(), and reuse it in ucs_sockaddr_str() function

File: src/ucp/core/ucp_listener.c [hunk 3]
```diff
@@ -246,6 +247,38 @@ ucp_listen(ucp_listener_h listener, const ucp_listener_params_t *params)
...
+    }
+}
+
+static void ucp_listener_vfs_show_port(void *obj, ucs_string_buffer_t *strb)
+{
+    ucp_listener_h listener   = obj;
+    struct sockaddr *sockaddr = (struct sockaddr*)&listener->sockaddr;
+    uint16_t port;
+
+    if (ucs_sockaddr_get_port(sockaddr, &port) == UCS_OK) {
```
Comment (yosefe) [inline src/ucp/core/ucp_listener.c:270]: print the error if failed

File: src/ucp/core/ucp_listener.c (hunk 2 above)
Comment (rakhmets) [reply src/ucp/core/ucp_listener.c:256]: @yosefe please find the changes here #6678

**Comment by rakhmets** [comment]
@yosefe could you please to review the changes

**Review by yosefe — APPROVED** [review]

**Pull request merged**